bin/install/upload_to_s3.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -x
+
+COSMOS_VERSION="0.4.0"
+
+cosmos_cli()
+{
+        result=$(echo -e "$COSMOS_VERSION\n$TAG_VERSION" | sed '/^$/d' | sort -nr | head -1)
+        [[ "$result" = "$TAG_VERSION" ]]
```

Comment:
What is this doing? Looks like we  doing a numeric short yet these are semver string. Please write a comment explaining why this work.